lib/rest_in_peace/template_sanitizer.rb

```diff
@@ -16,10 +17,14 @@ def url
       return @url if @url
       @url = @url_template.dup
       tokens.each do |token|
-        param = @params.delete(token.to_sym)
```

Comment:
I'm thinking about what the purpose of using `.delete` here was...
Could it be that it's in here so that the params which are in the path aren't appended as query params as well, e.g. `/path/:id` doesn't become `/path/1?id=1`?

@fho-wtag could you maybe check if this is the case now?